src/utils/form/validators/max/max.spec.ts

```diff
@@ -4,67 +4,69 @@ import { MaxValidator } from './max';
 describe('Max validator', () => {
     let formControl: FormControl<any>;
 
-    beforeAll(() => {
-        formControl = new FormControl<any>(
-            [MaxValidator('test', 3)]
-        );
-    });
+    describe('"any" type form ', () => {
+        test('it should return true if value is undefined', () => {
+            formControl = new FormControl<any>(
+                [MaxValidator(3)]
+            );
 
-    test('it should return true if value is undefined', async (done) => {
-        expect(formControl.value).toBe(undefined);
-        await formControl.validate();
-        expect(formControl.valid).toBe(true);
-        done();
+            formControl.validate();
+
+            expect(formControl.valid).toBe(true);
+        });
     });
 
     describe('number value', () => {
-        beforeAll(() => {
+
+        const MAX_VALUE: number = 3;
+
+        beforeEach(() => {
             formControl = new FormControl<number>(
-                [MaxValidator('test', 3)]
+                [MaxValidator(MAX_VALUE)]
             );
         });
 
-        test('it should return true if value is 0 and max value is higher', async (done) => {
+        test('it should return true if value is 0 and max value is higher', () => {
             formControl.value = 0;
-            await formControl.validate();
+            formControl.validate();
             expect(formControl.valid).toBe(true);
-            done();
         });
 
-        test('it should return false if value is 0 and max value is lower', async (done) => {
-            const formControlTest0: FormControl<number> = new FormControl<number>(
-                [MaxValidator('test', -1)],
+        test('it should return false if value is 0 and max value is lower', () => {
```

Comment:
Pk test au lieu de it?

Reply:
aucune idée, c'était déjà comme ca.. faudrait voir avec @guignol1981 ? j'ai pas checké, je pensais que c'était la méthode de modul.. -_-

Reply:
oopsie, pas de raison, je prendrai it dans le futur